babel.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   presets: [
     [
-      '@vue/app',
+      '@vue/cli-plugin-babel/preset',
```

Comment:
Why did we need to make this change?

Reply:
Per https://cli.vuejs.org/migrating-from-v3/#vue-cli-plugin-babel-vue-babel-preset-app

> In v3, the default babel preset used in babel.config.js is @vue/app.
 In v4, we moved it to the plugin, so now it's named as @vue/cli-plugin-babel/preset

> It is because that @vue/babel-preset-app is indeed an indirect dependency on the project. It works because of  npm's package hoisting. But potential problems could still occur if the project has multiple conflicting indirect dependencies of the same package, or if the package manager puts stricter constraints on the dependency resolution (e.g. yarn plug'n'play or pnpm). So we now moved it to the project's direct dependency (@vue/cli-plugin-babel) to make it more standard-compliant and less error-prone.